sites/website/src/docs/resources/cheat-sheet.md

```diff
@@ -0,0 +1,695 @@
+---
+id: cheat-sheet
+title: Cheat Sheet
+sidebar_label: Cheat Sheet
+custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/resources/cheat-sheet.md
+---
+
+# Cheat Sheet
+
+## Packages:
+
+### [@microsoft/fast-element](https://www.fast.design/docs/fast-element/getting-started)
+
+**A library that is a lightweight means to easily build performant, memory-efficient, standards-compliant Web Components.**
```

Comment:
Can we reword this? It reads a bit off, though I think I understand the intent. Specifically, I think "A library that is. lightweight means to..." could probably be reworked to be more easily consumed.

Reply:
@EisenbergEffect When you have a moment, I am not sure how to best reword this sentence.

Should I include some of the content from the intro page from the docs?

![Screen Shot 2021-12-15 at 2 18 28 PM](https://user-images.githubusercontent.com/35910788/146273747-dfd57ab8-758e-4f72-9632-58cd90fef783.png)

This sentence is found in the docs and in the repo readme.  Would it make sense to update in these two places as well?

![Screen Shot 2021-12-15 at 2 10 40 PM](https://user-images.githubusercontent.com/35910788/146272899-47f61ad8-9879-4c4c-a398-a54cc80ad4fb.png)

![Screen Shot 2021-12-15 at 2 10 30 PM](https://user-images.githubusercontent.com/35910788/146272934-7069fc94-a17f-4d96-9e1b-151b85d03356.png)

Reply:
Perhaps something like "A lightweight library for building performant, memory-efficient, standards-compliant Web Components."

Reply:
@EisenbergEffect Thank you - updated in the cheat sheet!